<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-30T16:19:08 IST" name="Suite" finished-at="2024-07-30T16:19:39 IST" duration-ms="30500">
    <groups>
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" name="submitOrder" class="rahulshettyacademy.tests.SubmitOrderTest"/>
      </group> <!-- Purchase -->
      <group name="ErrorHanding">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" name="LoginErrorValidation" class="rahulshettyacademy.tests.ErrorValidationsTest"/>
      </group> <!-- ErrorHanding -->
    </groups>
    <test started-at="2024-07-30T16:19:08 IST" name="Submit Order Test" finished-at="2024-07-30T16:19:37 IST" duration-ms="28850">
      <class name="rahulshettyacademy.tests.SubmitOrderTest">
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:09 IST" name="lauchApplication" finished-at="2024-07-30T16:19:16 IST" duration-ms="7169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:16 IST" name="submitOrder" data-provider="getData" finished-at="2024-07-30T16:19:21 IST" duration-ms="4546" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Saketa@1234, email=kattapraneeth94@gmail.com, products=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:21 IST" name="tearDown" finished-at="2024-07-30T16:19:21 IST" duration-ms="39" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:21 IST" name="lauchApplication" finished-at="2024-07-30T16:19:26 IST" duration-ms="5404" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:26 IST" name="submitOrder" data-provider="getData" finished-at="2024-07-30T16:19:30 IST" duration-ms="4307" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Saketa@1234, email=kattapraneeth98@gmail.com, products=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:30 IST" name="tearDown" finished-at="2024-07-30T16:19:30 IST" duration-ms="41" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:30 IST" name="lauchApplication" finished-at="2024-07-30T16:19:35 IST" duration-ms="5041" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:35 IST" name="OrderHistoryTest" finished-at="2024-07-30T16:19:37 IST" depends-on-methods="rahulshettyacademy.tests.SubmitOrderTest.submitOrder" duration-ms="1579" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.SubmitOrderTest@16c069df]" started-at="2024-07-30T16:19:37 IST" name="tearDown" finished-at="2024-07-30T16:19:37 IST" duration-ms="37" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshettyacademy.tests.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
    <test started-at="2024-07-30T16:19:08 IST" name="Error Validations Test" finished-at="2024-07-30T16:19:39 IST" duration-ms="30500">
      <class name="rahulshettyacademy.tests.ErrorValidationsTest">
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:08 IST" name="lauchApplication" finished-at="2024-07-30T16:19:16 IST" duration-ms="7902" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:16 IST" name="LoginErrorValidation" finished-at="2024-07-30T16:19:18 IST" duration-ms="1437" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at rahulshettyacademy.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:18 IST" name="tearDown" finished-at="2024-07-30T16:19:18 IST" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:18 IST" name="lauchApplication" finished-at="2024-07-30T16:19:23 IST" duration-ms="5401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:23 IST" name="LoginErrorValidation" finished-at="2024-07-30T16:19:24 IST" duration-ms="1387" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at rahulshettyacademy.tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:25 IST" name="tearDown" finished-at="2024-07-30T16:19:25 IST" duration-ms="33" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="lauchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:25 IST" name="lauchApplication" finished-at="2024-07-30T16:19:30 IST" duration-ms="5265" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:30 IST" name="ProductErrorValidation" finished-at="2024-07-30T16:19:39 IST" duration-ms="8360" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationsTest@14008db3]" started-at="2024-07-30T16:19:39 IST" name="tearDown" finished-at="2024-07-30T16:19:39 IST" duration-ms="37" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshettyacademy.tests.ErrorValidationsTest -->
    </test> <!-- Error Validations Test -->
  </suite> <!-- Suite -->
</testng-results>
